app/src/main/java/com/yml/womensafety/GetLocation.kt

```diff
@@ -0,0 +1,53 @@
+package com.yml.womensafety
+
+import android.content.Context
+import android.location.Geocoder
+import android.location.Location
+import android.util.Log
+import java.util.*
+
+class GetLocation(val context: Context) {
+    private var message = String()
```

Comment:
Do this instead `private var message = ""` or you can do it without using the global variable at all